packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -210,6 +210,12 @@ export interface IDetailsListProps extends React.Props<DetailsList>, IWithViewpo
    * Whether or not the selection zone should enter modal state on touch.
    */
   enterModalSelectionOnTouch?: boolean;
+
+  /**
+   * How to style rows when the cursor is hovering over a row.
+   * @default HoverMode.normal
+   */
+  hoverMode?: HoverMode;
```

Comment:
is there an opportunity of a 3rd value here? Can we turn this into a Boolean rather than an enum?

Reply:
I figured we may want to add other hover modes in the future. Boolean works too; I'll change it!